### ISSUE TITLE
Firefox, log in on Nexus not possible for "wrong password"

### ISSUE THREAD

**Issue by zcul** [Nexus-Mods/web-issues#517]
Only when running Firefox browser:
Everytime I try to log in on Nexus site an error says "wrong username or password".
Edit:
Log in only works for the Forum site, after that no log in status on the Nexus site.

**Comment by rscottexe** [comment]
I can't replicate this issue on my Firefox. Login is working fine for me.

**Comment by JimmyRJump** [comment]
I had something like this a couple of years ago (around the releaseof Fallout 4) and the main reason why I still browse the Nexus with Google Chrome although the issue went away somewhere along the line.

My issue at the time was slightly different in that I kept being sent around in a loop to the log-in page everytime I logged in.

Can't add anything helpful as I don't now what caused it and also not what solved it.
